scripts/pi/00-run-chroot.sh

```diff
@@ -14,12 +14,18 @@ export GIT_SHA='$(git rev-parse --short HEAD)'
 # Enable root login
 #sed -i 's/.*PermitRootLogin.*/PermitRootLogin yes/g' /etc/ssh/sshd_config
 
-echo 'Disabling Bluetooth...'
-systemctl disable bluetooth.service
-cat >> /boot/config.txt <<EOF
-enable_uart=1
-dtoverlay=disable-bt
-EOF
+# No longer disabling bluetooth to support tilt(s)
+# echo 'Disabling Bluetooth...'
+# systemctl disable bluetooth.service
+# cat >> /boot/config.txt <<EOF
+# enable_uart=1
```

Comment:
I think you'd still want `enable_uart=1`

Reply:
oops, I wasn't sure. I'll put it back.

I will test this stuff on my raspberry pi locally since I have an existing install with this enabled, but do you have any pointers for doing a full image test?

Reply:
Yeah - jsut follow the readme (https://github.com/chiefwigms/picobrew_pico/blob/master/scripts/pi/_readme.txt).  That works fine on WSL2 in windows, should work natively in Ubuntu, and I'm pretty sure @tmack8001 runs it fine w/ his shiny mac

Reply:
Uncomment/fix pathing @ `#cp <Picobrewserver Root>/picobrew_pico/scripts/pi/00-run-chroot.sh stage2/99-picobrewserver-setup`

Reply:
ok, thanks.